client/grpc/tmservice/service.go

```diff
@@ -192,16 +215,26 @@ func (s queryServer) GetNodeInfo(ctx context.Context, req *GetNodeInfoRequest) (
 	return &resp, nil
 }
 
+func (s queryServer) ABCIQuery(ctx context.Context, req *ABCIQueryRequest) (*ABCIQueryResponse, error) {
```

Comment:
this doesn't get routed to tendermint right?

Reply:
Correct! It goes right to the app :)